content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -197,14 +198,19 @@ Elimination of resources or fields requires following the
 
 Kubernetes makes a strong commitment to maintain compatibility for official Kubernetes APIs
 once they reach general availability (GA), typically at API version `v1`. Additionally,
-Kubernetes keeps compatibility even for _beta_ API versions wherever feasible:
-if you adopt a beta API you can continue to interact with your cluster using that API,
-even after the feature goes stable.
+Kubernetes maintains compatibility with data persisted via _beta_ API versions of official Kubernetes APIs,
+and ensures that data can be converted and accessed via GA API versions when the feature goes stable.
```

Comment:
This implies "forever" which is not true, but I'm not sure how to fix it

Reply:
```suggestion
and ensures that data can (for a reasonable amount of time) be converted and accessed via GA API versions when the feature goes stable.
```

maybe

Reply:
I don't think this is changing current state... this is saying a beta → GA data conversion must exist, which will always be true. The conversions may not eternally be compiled into the API server if we ever get our server data migration story together, but any change to remove those would need to seamlessly ensure conversion of beta data was completed before removal

Reply:
In isolation I still think this is misleading, but the next paragraph explains it, so I don't think it has to change.